### PR TITLE
Modified API to reuse the grpc stub

### PR DESCRIPTION
**Pull request by bigspider**
It looks like reusing the channel/stub is the correct way of making repeated grpc requests, as channels already support multiplexing concurrent requests on them.

This PR recycles the `stub` for the `API` class, and adds missing attribute docs for `_RPC`.

Closes: #193

### CODE REVIEW

File: teos/api.py [hunk 1]
```diff
@@ -99,6 +99,8 @@ class API:
     Attributes:
         logger (:obj:`Logger <teos.logger.Logger>`): The logger for this component.
         app: The Flask app of the API server.
+        channel (:obj:`grpc.Channel`): The :obj:`Channel` object.
+        stub: The rpc client stub.
```
Comment (sr-gi) [inline teos/api.py:103]: Add type in docstring

File: teos/api.py [hunk 2]
```diff
@@ -107,6 +109,8 @@ def __init__(self, inspector, internal_api_endpoint):
         self.app = Flask(__name__)
         self.inspector = inspector
         self.internal_api_endpoint = internal_api_endpoint
+        self.channel = grpc.insecure_channel(internal_api_endpoint)
```
Comment (sr-gi) [inline teos/api.py:112]: Do we need to store the channel?

File: teos/rpc.py [hunk 3]
```diff
@@ -67,6 +67,10 @@ class _RPC(TowerServicesServicer):
     Args:
         internal_api_endpoint (:obj:`str`): the endpoint where to reach the internal (gRPC) api.
         logger (:obj:`Logger <teos.logger.Logger>`): the logger for this component.
+
+    Attributes:
+        channel (:obj:`grpc.Channel`): the Channel object
+        stub (:obj:`TowerServicesStub`): the rpc client stub
```
Comment (sr-gi) [inline teos/rpc.py:73]: Add type to docs

**Review by sr-gi — CHANGES_REQUESTED** [review]
Minor comments.

In case of removing `self.channel`, that may apply to `rpc.py`, `api.py` and `teos_cli.py`.

File: teos/api.py (hunk 2 above)
Comment (bigspider) [reply teos/api.py]: Removed (also in rpc and teos_cli) in 5988290.

File: teos/api.py (hunk 1 above)
Comment (bigspider) [reply teos/api.py]: Added in 5988290.

**Review by sr-gi — APPROVED** [review]

**Pull request merged**